Models/ObjectsFactoryModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EntityManager.Models
+{
+    public class ObjectsFactoryModel
+    {
+        public string VbName { get; set; }
+        public string Description { get; set; }
+        public List<FunctionModel> Functions { get; set; } = new List<FunctionModel>();
+
+        public int GetMaxFunctionNameLenght() => Functions.Max(f => f.VbaName.Length);
+        public int GetMaxFunctionDeclarationLenght() => Functions.Max(f => f.VbaDeclaration.Length);
```

Comment:
maybe these could be only getters? without setter

Reply:
it is used only once in whole app, doesnt really matter much.